tools/fee-measurer/scripts/measure.ts

```diff
@@ -0,0 +1,164 @@
+      const ordinaryGas = await feeMeasurer.useGasOrdinaryGasUsed(size);
+      const receipt = (await (await feeMeasurer.useGas(size, overrides)).wait())!;
+      referenceBlock = receipt.blockNumber;
+      const balanceAfter = await ethers.provider.getBalance(await signer.getAddress(), referenceBlock);
+
+      const result = {
+        size,
+        ordinaryGas,
+        reportedGasUsed: receipt.gasUsed,
+        reportedGasPrice: receipt.gasPrice,
+        weiUsed: balanceBefore - balanceAfter,
+        ethUsedFmt: ethers.formatEther(balanceBefore - balanceAfter),
+      };
+
+      console.log(result);
+      results.push(result);
+    }
+
+    const firstResult = results[0];
+    const lastResult = results.at(-1)!;
+
+    ordinaryGasPrice =
+      (lastResult.weiUsed - firstResult.weiUsed) /
+      (lastResult.ordinaryGas - firstResult.ordinaryGas);
+
+    const weiUsedRelErrors = results.map((result) => {
+      const prediction = (firstResult.weiUsed
+        + ordinaryGasPrice * (result.ordinaryGas - firstResult.ordinaryGas));
+
+      return Number(result.weiUsed - prediction) / Number(result.weiUsed);
+    });
+
+    console.log({ ordinaryGasPrice, weiUsedRelErrors });
+  }
+
+  {
+    console.log();
+    console.log();
+    console.log("Measuring calldata price...");
+    console.log("===========================");
+    console.log();
+
+    const results = [];
+
+    for (const size of [0n, 50n, 100n, 150n]) {
+      console.log({ size });
+
+      const balanceBefore = await ethers.provider.getBalance(await signer.getAddress(), referenceBlock);
+      const ordinaryGas = await feeMeasurer.fallbackOrdinaryGasUsed(size);
+      const receipt = (await (await (feeMeasurer.fallback!)({
+        ...overrides,
+        data: generateBytes(Number(size)),
+      })).wait())!;
+      referenceBlock = receipt.blockNumber;
+
+      const balanceAfter = await ethers.provider.getBalance(await signer.getAddress(), referenceBlock);
+      const weiUsed = balanceBefore - balanceAfter;
+      const weiUsedForOrdinaryGas = ordinaryGasPrice * ordinaryGas;
+      const weiUsedExtra = weiUsed - weiUsedForOrdinaryGas;
+
+      const result = {
+        size,
+        ordinaryGas,
+        reportedGasUsed: receipt.gasUsed,
+        reportedGasPrice: receipt.gasPrice,
+        weiUsed,
+        weiUsedForOrdinaryGas,
+        weiUsedExtra,
+        ethUsed: ethers.formatEther(weiUsed),
+        ethUsedForOrdinaryGas: ethers.formatEther(weiUsedForOrdinaryGas),
+        ethUsedExtra: ethers.formatEther(weiUsedExtra),
+      };
+
+      console.log(result);
+      results.push(result);
+    }
+
+    const firstResult = results[0];
+    const lastResult = results.at(-1)!;
+
+    const baselineExtraWei = firstResult.weiUsedExtra;
+    const extraWeiPerByte = (lastResult.weiUsedExtra - firstResult.weiUsedExtra) / (lastResult.size - firstResult.size);
+
+    const weiUsedRelErrors = results.map((result) => {
```

Comment:
The calculation for `weiUsedRelErrors` here looks slightly different to the first one used when measuring ordinary gas price, why is that?